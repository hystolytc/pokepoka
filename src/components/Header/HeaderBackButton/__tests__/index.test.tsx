import React from 'react'
import '@testing-library/jest-dom/extend-expect'
import { render } from '@testing-library/react'
import { HeaderBackButton } from '../index'

const describeComponent = '<HeaderBackButton /> test case'

describe(describeComponent, () => {
  const { container } = render(
    <HeaderBackButton
      title='Title test'
      goBack={() => {}}
      goMyPokemon={() => {}} />
  )

  test(`Render ${describeComponent} correctly`, () => {
    expect(container).toMatchSnapshot()
  })
})