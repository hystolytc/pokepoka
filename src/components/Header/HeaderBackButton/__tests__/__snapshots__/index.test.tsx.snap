// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderBackButton /> test case Render <HeaderBackButton /> test case correctly 1`] = `
<div>
  <header
    class="css-1vo74qe"
  >
    <div
      class="css-2wlbqr"
    >
      <a
        class="css-kujwy0"
      >
        Back
      </a>
      <span>
        Title test
      </span>
    </div>
  </header>
</div>
`;
